contracts/transfer/tests/phoenix.rs

```diff
@@ -965,6 +966,143 @@ fn contract_withdraw() {
     );
 }
 
+/// Converting moonlight DUSK into phoenix DUSK with a phoenix transaction
+/// should fail.
+#[test]
+fn convert_to_phoenix_fails() {
+    const CONVERSION_VALUE: u64 = dusk(10.0);
+
+    let rng = &mut StdRng::seed_from_u64(0xfeeb);
+
+    let phoenix_sender_sk = PhoenixSecretKey::random(rng);
+    let phoenix_sender_vk = PhoenixViewKey::from(&phoenix_sender_sk);
+    let phoenix_sender_pk = PhoenixPublicKey::from(&phoenix_sender_sk);
+
+    let phoenix_change_pk = phoenix_sender_pk.clone();
+
+    let moonlight_sk = AccountSecretKey::random(rng);
+    let moonlight_pk = AccountPublicKey::from(&moonlight_sk);
+
+    let mut session = &mut instantiate::<1>(rng, &phoenix_sender_sk);
+
+    // Add the conversion value to the moonlight account
+    session
+        .call::<_, ()>(
+            TRANSFER_CONTRACT,
+            "add_account_balance",
+            &(moonlight_pk, CONVERSION_VALUE),
+            GAS_LIMIT,
+        )
+        .expect("Inserting genesis account should succeed");
+
+    // make sure the moonlight account doesn't own any funds before the
+    // conversion
+    let moonlight_account = account(&mut session, &moonlight_pk)
+        .expect("Getting account should succeed");
+    assert_eq!(
+        moonlight_account.balance, CONVERSION_VALUE,
+        "The moonlight account should have the max gas before the transaction"
```

Comment:
```suggestion
        "The moonlight account should have the conversion value before the transaction"
```
Same as with the comment above?